docs/developers/documentation/index.md

```diff
@@ -21,10 +38,16 @@ existing documents that could be expanded to include the content you think is la
 
 Examples of documents you might want to contribute are:
 
-- **Explanations** (in `napari/docs/guides`): in depth content about napari architecture, development choices and some complex features
-- **Tutorials** (in `napari/docs/tutorials`): detailed, reproducible step by step guides, usually combining multiple napari features to complete a potentially complex task
-- **How-tos** (in `napari/docs/howtos/`): simple step by step guides demonstrating the use of common features
-- **Getting started** (in `napari/docs/tutorials/fundamentals`): these documents are a mix of tutorials and how-tos covering the fundamentals of installing and working with napari for beginners
+- [**Explanations**](/docs/guides/index) (in [`napari/docs/guides`](https://github.com/napari/napari/tree/main/docs/guides)):
```

Comment:
No idea why these links are not working. I have tried both absolute and relative paths, and with and without file extension. From https://myst-parser.readthedocs.io/en/latest/develop/_changelog.html#markdown-link-resolution-improvements and https://github.com/napari/napari/pull/4787 they should work?

Reply:
Any ideas @melissawm

Reply:
Nevermind, worked it out!